src/lib/BufferedDuplex.ts

```diff
@@ -1,23 +1,37 @@
 import { Duplex, type Transform } from 'readable-stream'
 import { Buffer } from 'buffer'
-import { type IClientOptions } from './client'
+import { type IClientOptions } from './client.js'
+
+interface WritevChunksParameters {
+	chunk: unknown
+	encoding: string
+}
```

Comment:
I elected to create this very simple interface to limit cognitive load on function signature. It also simplified later typing.